wolfssl/wolfcrypt/settings.h

```diff
@@ -882,6 +882,13 @@ extern void uITRON4_free(void *p) ;
     #define TFM_TIMING_RESISTANT
 #endif
 
+#ifdef WOLFSSL_USE_FLASHMEM
+    #define FLASH_QUALIFIER __flash
```

Comment:
Can you add a note here about the type of compiler this is for?

Reply:
Yep, can do